pages/blog/react-as-generator-engine.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Speed your template development with React"
+date: 2021-01-25T00:07:00+01:00
+type: Engineering
+featured: true
+tags:
+  - Tools
+  - Generator
+  - Templating
+  - React
+  - Nunjucks
+cover: /img/posts/react-as-generator-engine/react-as-generator-engine.webp
+authors:
+  - name: Jonas Lagoni
+    photo: /img/avatars/jonaslagoni.webp
+    link: https://github.com/jonaslagoni
+    byline: AsyncAPI Core Team Member
+  - name: Maciej Urbańczyk
+    photo: /img/avatars/maciejurbanczyk.webp
+    link: https://github.com/magicmatatjahu
+    byline: AsyncAPI Core Team Member
+---
+
+React permanently changing the way how developers write web-apps. Personally, we love React and we tried to integrate it as a template rendering engine in our [Generator](https://github.com/asyncapi/generator). This post is a short introduction for developers who write/want to write templates for AsyncAPI specification using React. It also includes a comparison to the Nunjucks, which is default renderer in the Generator.
```

Comment:
```suggestion
React is permanently changing the way how developers write web-apps. Personally, we love React and knew it would solve many pain points we where faced with Nunjucks. Therefore in the last [cycle](https://github.com/asyncapi/shape-up-process/issues/1) we integrated it as a template rendering engine into our [Generator](https://github.com/asyncapi/generator). This post is a short introduction for developers who write/want to write templates for AsyncAPI specification using React. It also includes a comparison to the default Nunjucks renderer.
```